peachjam/js/components/FindDocuments/SearchResult.vue

```diff
@@ -95,6 +95,14 @@
         </div>
       </div>
     </div>
+    <div v-if="showFeedbackCard" class="card mt-4">
```

Comment:
I suggest making this a component in its own right, since it shows up in a few places.